packages/toast/test/toast.test.ts

```diff
@@ -56,15 +56,82 @@ describe('Toast', () => {
         it(`loads - [variant="${variant}"]`, async () => {
             const el = await fixture<Toast>(
                 html`
-                    <sp-toast variant=${variant} open>
+                    <sp-toast variant=${variant} open label="testLabel">
+                        This toast is of the \`${variant}\` variant.
+                    </sp-toast>
+                `
+            );
+            await elementUpdated(el);
+            await expect(el.getAttribute('label')).to.equal('testLabel');
+            await expect(el).to.be.accessible();
+        });
+
+        it(`customized label for sp-icon for [variant="${variant}"]`, async () => {
+            const el = await fixture<Toast>(
+                html`
+                    <sp-toast variant=${variant} open label="testLabel">
                         This toast is of the \`${variant}\` variant.
                     </sp-toast>
                 `
             );
 
+            const getIconType = (): string => {
+                switch (variant) {
+                    case 'info':
+                        return 'sp-icon-info';
+                    case 'negative':
+                    case 'error': // deprecated
+                    case 'warning': // deprecated
+                        return 'sp-icon-alert';
+                    case 'positive':
+                        return 'sp-icon-checkmark-circle';
+                    default:
+                        return '';
+                }
+            };
+
             await elementUpdated(el);
+            const icon = el.shadowRoot.querySelector(
+                getIconType()
+            ) as HTMLElement;
+            await expect(icon.getAttribute('label')).to.equal('testLabel');
+        });
 
-            await expect(el).to.be.accessible();
+        it(`default label for sp-icon for [variant="${variant}"]`, async () => {
+            const el = await fixture<Toast>(
+                html`
+                    <sp-toast variant=${variant} open>
+                        This toast is of the \`${variant}\` variant.
+                    </sp-toast>
+                `
+            );
+
+            let iconType;
+            let defaultLabel;
+            switch (variant) {
+                case 'info':
+                    iconType = 'sp-icon-info';
+                    defaultLabel = 'Information';
+                    break;
+                case 'negative':
+                case 'error': // deprecated
+                case 'warning': // deprecated
+                    iconType = 'sp-icon-alert';
+                    defaultLabel = 'Error';
+                    break;
+                case 'positive':
+                    iconType = 'sp-icon-checkmark-circle';
+                    defaultLabel = 'Success';
+                    break;
+                default:
+                    iconType = '';
+                    defaultLabel = '';
+                    break;
+            }
```

Comment:
Some of this can be simplified via `class="type"` being on the icon element. The rest of the data feels like it should be externalized some how, but I don't have a good suggestion on cleaning it up just now. Maybe an map of some sort outside of the test case?